packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/app/my_app.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:{{project_name}}/base/app/environment_config.dart';
+import 'package:{{project_name}}/base/routers/router.gr.dart' as router;
+import 'package:{{project_name}}/base/utils/providers.dart';
+
+/// This widget is the root of your application.
+class MyApp extends StatelessWidget {
```

Comment:
@DDavidPrime, please get the name of the app from the command param --project-name